lib/mayaUsd/utils/diagnosticDelegate.cpp

```diff
@@ -68,7 +74,8 @@ static MString _FormatDiagnostic(const TfDiagnosticBase& d)
         d.GetContext().GetFunction(),
         d.GetContext().GetLine(),
         d.GetContext().GetFile());
```

Comment:
I wonder if we should strip off the path from this and leave only the filename? Isn't part of the problem that we are leaking our paths from the build VMs. I would again suggest that in Debug you leave it alone, but in RelWithDebInfo you strip it. Our Debug builds are never shipped.

Reply:
But a user can enable the new env var and then they will see all the paths from our VMs.

Reply:
Can we not create a simple regex to strip everything before maya-usd/src?